merge_checks.py

```diff
@@ -4,46 +4,38 @@
 from typing import Tuple
 
 
+def _run_process(command: str) -> str:
+    subprocess.run(
```

Comment:
Refucktoring...

Reply:
Was soll mir das sagen?

Reply:
Vermutlich sind mehrere Interpretationen möglich, allerdings die gemeinte war, dass du (vermutlich unabsichtlich) der Code bei dem Refactoring kaputtgemacht hast, indem du in diese Funktion die Ergebnisse der Ausführung eben nicht zurückgibst, auch wenn das der Typ widerspricht:

<img width="739" alt="Screenshot 2021-07-28 at 14 03 40" src="https://user-images.githubusercontent.com/607401/127319118-eedbe565-2ebd-4bea-a39f-52f6e5023658.png">

Deswegen sind die Merge Checks kaputt.

Mit Tests anpassen meinte ich, dass ich das unzufriedenstellend finde, dass die Tests sind aber trotzdem durchgelaufen und dir dazu kein Hinweis gegeben haben.

Reply:
Das wird vmtl. daran liegen, dass die Tests gar nicht bis auf die `subprocess.run`-Ebene runtergehen, sondern die verwendenden Funktionen mocken.

Reply:
Genau, also überlege dich bitte, wie man zumindest das Happy-Path Test so anpasst, dass der sicherstellt, dass `subprocess.run` mit sinnvollen Argumenten aufgerufen wurde und über Side Effects Mock Daten zurückgegeben hat, die korrekt verarbeitet wurden, statt einfach die Abdeckung runterdrehen.